### PR TITLE
WIP Feat/add nav bar

### LINKED ISSUE #18: Create navigation bar with signup/login button

### PR DESCRIPTION
**Pull request by obay-mardini**
Please don't merge this pull request closes #18

### CODE REVIEW

File: client/src/containers/NavBar.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	background-color:rgb(0, 188, 212);
+	margin: 0;
+	padding: 0;
+	-webkit-box-shadow:0px 2px 1px rgba(50, 50, 50, 0.22);
+	-moz-box-shadow:0px 2px 1px rgba(50, 50, 50, 0.22);
+	box-shadow:0px 2px 1px rgba(50, 50, 50, 0.22);
+}
+.nav li {
+	text-align:center;
+	font-family: open sans;
```
Comment (gzeegz) [inline client/src/containers/NavBar.css]: Let's take off the font here. We want a consistent font throughout the app.

File: client/src/containers/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	<div>
+		<ul className="nav">
+	    <li className="active"><Link to="/">Home</Link></li>
+	    <li><Link to="/meals">Meals</Link></li>
+			<li><Link to="/post-new-meal">New Meal</Link></li>
+			<li className="nav-auth"><Link to="/sign-up-form">Signup</Link></li>
+			<li className="nav-auth"><Link to="/log-in-form">Login</Link></li>
+		</ul>
+	</div>
+);
```
Comment (gzeegz) [inline client/src/containers/NavBar.js]: Indentation needs to be fixed.

**Review by gzeegz — COMMENTED** [review]
The indentation and spacing of the CSS files are not consistent.

**Comment by obay-mardini** [comment]
The navigation bar is ready now.... please feel free to review it

**Review by gzeegz — APPROVED** [review]
LGTM

**Pull request merged**